devtools/maven/src/main/java/io/quarkus/maven/CreateUtils.java

```diff
@@ -81,6 +82,14 @@ static QuarkusPlatformDescriptor resolvePlatformDescriptor(final String bomGroup
             version = resolvePluginInfo(CreateUtils.class).getVersion();
         }
 
+        if((bomGroupId==null && bomArtifactId==null && bomVersion==null) || bomArtifactId=="quarkus-universe-bom"){
+            String baseVersion = resolvePluginInfo(CreateUtils.class).getVersion();
+            DefaultArtifactVersion pluginVersion= new DefaultArtifactVersion(baseVersion);
+            int majorVer=pluginVersion.getMajorVersion();
+            int minorVer=pluginVersion.getMinorVersion();
+            version = "[" + majorVer + "." + minorVer + "-snapshot, " + majorVer + "." + (minorVer + 1) + ")";
```

Comment:
It should be `-SNAPSHOT`

Reply:
-snapshot is ok, i suggested that actually